### DISCUSSION TITLE
Found this software on my computer, how did it get there if in testing?

### DISCUSSION THREAD

**Discussion by SlyCooperking** [hughsie/passim#75] [General]
Hello, Found this software via netstat. Deployed from Fedora Silverblue rpm-ostree must have been an update Hash f19c609b1e76b13db38fcd9d672a52e40f52fbe791ab920e8318c7ec6877055a.0 

It looks like this is required by the following software as of 10/07/2024
dnf repoquery --requires passim
Fedora 38 - x86_64                                                         34 kB/s | 6.0 kB     00:00    
Fedora 38 openh264 (From Cisco) - x86_64                                  6.3 kB/s | 989  B     00:00    
Fedora Modular 38 - x86_64                                                 42 kB/s | 5.9 kB     00:00    
Fedora 38 - x86_64 - Updates                                               46 kB/s | 5.9 kB     00:00    
Fedora Modular 38 - x86_64 - Updates                                       37 kB/s | 5.8 kB     00:00    
/bin/sh
glib2(x86-32) >= 2.45.8
glib2(x86-64) >= 2.45.8
libc.so.6(GLIBC_2.34)
libc.so.6(GLIBC_2.34)(64bit)
libgcc_s.so.1
libgcc_s.so.1()(64bit)
libgcc_s.so.1(GCC_3.0)
libgcc_s.so.1(GCC_3.0)(64bit)
libgcc_s.so.1(GCC_3.3.1)
libgcc_s.so.1(GCC_3.3.1)(64bit)
libgio-2.0.so.0
libgio-2.0.so.0()(64bit)
libglib-2.0.so.0
libglib-2.0.so.0()(64bit)
libgnutls.so.30
libgnutls.so.30()(64bit)
libgnutls.so.30(GNUTLS_3_4)
libgnutls.so.30(GNUTLS_3_4)(64bit)
libgnutls.so.30(GNUTLS_3_6_0)
libgnutls.so.30(GNUTLS_3_6_0)(64bit)
libgobject-2.0.so.0
libgobject-2.0.so.0()(64bit)
libpassim.so.1
libpassim.so.1()(64bit)
libpassim.so.1(LIBPASSIM_0.1.0)
libpassim.so.1(LIBPASSIM_0.1.0)(64bit)
libpassim.so.1(LIBPASSIM_0.1.2)
libpassim.so.1(LIBPASSIM_0.1.2)(64bit)
libsoup-3.0.so.0
libsoup-3.0.so.0()(64bit)
passim-libs(x86-32) = 0.1.5-1.fc38
passim-libs(x86-64) = 0.1.5-1.fc38
rtld(GNU_HASH)

Why is it a dependency for /bin/sh?

**Comment by hughsie** [comment]
> dnf repoquery --requires passim

No, this is what passim itself requires. It's probably being included because fwupd optionally uses it. See https://fedoraproject.org/wiki/Changes/Passim_P2P_Metadata for the Fedora change.
